src/communication/xcConnection.ts

```diff
@@ -3,6 +3,7 @@ import { ApiConfiguration } from "configuration/apiConfiguration";
 import { DefaultApiConfigurationParser } from "configuration/apiConfigurationParser";
 
 export interface Connection {
+    getModel(xcApiName: string, serverUrl: string, getModelListener: (model: any) => void);
```

Comment:
1. is it possible to be more precise on "any"?
2. getModelListener... looking at the other methods... shouldn't it be modelListener ?